scripts/ingests/utils.py

```diff
@@ -19,32 +20,19 @@
 import numpy as np
 
 warnings.filterwarnings("ignore", module='astroquery.simbad')
+logging.basicConfig(format='%(asctime)s %(levelname)s:%(message)s',
+                    datefmt='%m/%d/%Y %I:%M:%S %p')
 
```

Comment:
A good addition here is something like `logger = logging.getLogger('SIMPLE')`, you would then call logger.debug, etc instead of logging and if you need to refer to the logging level, you don't need the call to get the logger.